### ISSUE TITLE
DNNLConfig.cmake couldn't be found while building from source using cmake for CPU backend

### ISSUE THREAD

**Issue by Kevinpsk** [facebookresearch/flashlight#414]
### My platform:
Ubuntu 18.04 from WSL on Windows 10.

### Question
As suggested in the title, I met the following error while I was trying to build flashlight from source using cmake for the CPU backend only. I have installed all of the dependences given by the installation instructions, and for the intel MKL and DNNL, I installed the binary distributions (**intel oneAPI base Toolkit**) from the official [intel website](https://software.intel.com/content/www/us/en/develop/tools/oneapi/base-toolkit/download.html), into "/usr/opt/intel" and the build script can indeed find the intel MKL library, but I am not sure why it complained about the DNNL cmake files. 

I have a feeling this can be solved quite easily by changing some path maybe, but I am just not sure how. So can anyone help me? Thanks a lot for your help in advance. 

```
CMake Error at flashlight/fl/autograd/CMakeLists.txt:20 (find_package):
Could not find a package configuration file provided by "DNNL" (requested
version 2.0) with any of the following names:

  DNNLConfig.cmake
  dnnl-config.cmake

Add the installation prefix of "DNNL" to CMAKE_PREFIX_PATH or set
"DNNL_DIR" to a directory containing one of the above files.  If "DNNL" 
provides a separate development package or SDK, be sure it has been
installed.
Call Stack (most recent call first):
flashlight/fl/CMakeLists.txt:52 (include)
CMakeLists.txt:243 (include)
````

**Comment by tlikhomanenko** [comment]
You need to add something like `-DNNL_DIR=/opt/dnnl/dnnl_lnx_2.0.0_cpu_iomp/lib/cmake/dnnl` depending where you have installed the ondnn.

**Comment by Kevinpsk** [comment]
Hi @tlikhomanenko,

Thanks for your help. I found the colab install script and in there the MKL and DNNL are installed separately. But I was installing the oneAPI toolkit which installed both MKL and DNNL into the same directory of /opt/intel/oneapi. 

I followed the instructions in the colab install script and it now works, although I installed old versions of intel MKL and DNNL. I think if I set the paths correctly as you suggested to the directories inside /opt/intel/oneapi, the build would work as well with the most up-to-date MKL and DNNL. I will try this maybe later, but for now everything works now. And I am closing this question. 
Cheers,
Kevin
